pkg/scheduler/cache/cache.go

```diff
@@ -915,6 +915,12 @@ func (sc *SchedulerCache) taskUnschedulable(task *schedulingapi.TaskInfo, reason
 func (sc *SchedulerCache) deleteJob(job *schedulingapi.JobInfo) {
```

Comment:
There are some other calls who called `deleteJob` method，do we also need replace it with `retryDeleteJob`?

Reply:
Where?

Reply:
like https://github.com/volcano-sh/volcano/blob/2c5c1f9602eb3cae9645c3d5e299c6921f0685ff/pkg/scheduler/cache/event_handlers.go#L206

Reply:
Maybe you misunderstood the purpose of this PR. The purpose of this PR is to avoid triggering the rate limiter when the job cache is deleted for the first time, so there is no need to modify this place.

Reply:
got that！thanks for your contribution.